pkg/proc/fncall.go

```diff
@@ -1243,6 +1285,8 @@ func debugCallProtocolReg(archName string, version int) (uint64, bool) {
 			return regnum.ARM64_X0 + 20, true
 		}
 		return 0, false
+	case "ppc64le":
```

Comment:
This should check the version as well

Reply:
Thanks i had missed this ! I presume by version this indicates the version of debugCall such as debugCallV2